components/Articles.vue

```diff
@@ -75,6 +68,13 @@ export default {
   },
 
   methods: {
+    linkMedium (link) {
```

Comment:
I think this method name can be better, like openMediumArticle, openMediumPage, ...

Reply:
@minapham17 replaced by handleArticleClick